bitnami/postgresql/templates/secrets.yaml

```diff
@@ -1,16 +1,23 @@
 apiVersion: v1
 kind: Secret
 metadata:
-  name: {{ template "fullname" . }}
+  name: {{ template "postgresql.fullname" . }}
   labels:
-    app: {{ template "fullname" . }}
-    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    app: {{ template "postgresql.name" . }}
+    chart: {{ template "postgresql.chart" . }}
     release: "{{ .Release.Name }}"
     heritage: "{{ .Release.Service }}"
 type: Opaque
 data:
   {{ if .Values.postgresqlPassword }}
-  postgresql-password: {{ default "" .Values.postgresqlPassword | b64enc | quote }}
+  postgresql-password: {{ .Values.postgresqlPassword | b64enc | quote }}
```

Comment:
What happens if `.Values.postgresqlPassword` is not defined?

Reply:
Then a 10 character alphanumeric password is generated:

```
  {{ if .Values.postgresqlPassword }}
  postgresql-password: {{ .Values.postgresqlPassword | b64enc | quote }}
  {{ else }}
  postgresql-password: {{ randAlphaNum 10 | b64enc | quote }}
  {{ end }}
```